frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -0,0 +1,150 @@
+{
+  "Disable": "Disable",
+  "Enable": "Enable",
+  "Name": "Name",
+  "Publisher": "Publisher",
+  "Packages": "Packages",
+  "Package not found": "Package not found",
+  "Cannot create a Subscription to a non-existent package.": "Cannot create a Subscription to a non-existent package.",
+  "Status": "Status",
+  "Availability": "Availability",
+  "Endpoint": "Endpoint",
+  "Registry poll interval": "Registry poll interval",
+  "# of Operators": "# of Operators",
+  "Disabled": "Disabled",
+  "Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.": "Operators provided by this source will not appear in OperatorHub and any operators installed from this source will not receive updates until this source is re-enabled.",
+  "Enable source": "Enable source",
+  "Create {{item}}": "Create {{item}}",
+  "Edit Subscription": "Edit Subscription",
+  "Uninstall Operator": "Uninstall Operator",
+  "Cannot update": "Cannot update",
```

Comment:
should `Succeeded` or `Failed` status be translated? 
<img width="1488" alt="Screen Shot 2020-12-15 at 6 47 43 PM" src="https://user-images.githubusercontent.com/12692381/102205508-3ebf2500-3f06-11eb-8762-6e64b65f75af.png">

Reply:
We can't localize dynamic k8s resource properties. The "Succeeded" and "Failed" statuses that you see in that column come the CSV `status.phase` which we can't localize.  The "Cannot update" and "CatalogSource was removed" statuses are a static strings that are defined in our code, so we can localize those. I will address localization of the  second line where possible.

Reply:
Ok, got it